pom.xml

```diff
@@ -0,0 +1,52 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>gcp-java-tools-lib</artifactId>
+  <version>0.3-SNAPSHOT</version>
+
+  <name>Google Cloud Java management client library</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.target.version>1.9.34</appengine.target.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
```

Comment:
Oh that makes sense that you have to include the sdk.root.home or whatever when you're calling AppCfg.Main...  I'm exploring calling stage from the cloud-sdk included libraries, so maybe there's a chance we wont need this